cmd/notary-server/config.go

```diff
@@ -25,7 +25,7 @@ import (
 	"github.com/theupdateframework/notary/tuf/signed"
 	"github.com/theupdateframework/notary/utils"
 	"golang.org/x/net/context"
-	"gopkg.in/gorethink/gorethink.v3"
+	gorethink "gopkg.in/rethinkdb/rethinkdb-go.v6"
```

Comment:
No they renamed the package name from `gorethink` to `rethinkdb`

https://gopkg.in/rethinkdb/rethinkdb-go.v3
https://gopkg.in/rethinkdb/rethinkdb-go.v6

so figured the path of least resistance was to alias it, also because there is another `rethinkdb` package in this code that it caused conflict with in some of the files.

Reply:
oh! for some reason I only saw the `-go` suffix 🤦‍♂️